internal/daysteps/daysteps.go

```diff
@@ -1,13 +1,63 @@
 package daysteps
 
+import (
+	"errors"
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/Yandex-Practicum/tracker/internal/personaldata"
+	"github.com/Yandex-Practicum/tracker/internal/spentenergy"
+)
+
+var (
+	ErrSplit             = errors.New("ошибка разбиения строки")
+	ErrConvertation      = errors.New("ошибка конвертации")
+	ErrStepsIncorrect    = errors.New("ошибка некорректное число шагов")
+	ErrParsDuration      = errors.New("ошибка парсинга времени")
+	ErrDurationIncorrect = errors.New("ошибка некорректная продолжительность")
+)
+
 type DaySteps struct {
-	// TODO: добавить поля
+	Steps    int
+	Duration time.Duration
+	personaldata.Personal
 }
 
 func (ds *DaySteps) Parse(datastring string) (err error) {
-	// TODO: реализовать функцию
+	parts := strings.Split(datastring, ",")
+	if len(parts) != 2 {
+		return ErrSplit
+	}
+	//Первая часть
+	steps, err := strconv.Atoi(parts[0])
+	if err != nil {
+		return ErrConvertation
```

Comment:
Теряешь сообщение с ошибкой
```suggestion
		return fmt.Errorf("invalid steps: %w", err)
```